script.js

```diff
@@ -6,7 +6,25 @@ function setup() {
 
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+
+  return episodeList.map(episode => {
+    const div = document.createElement("div");
+    div.classList.add("episode");
```

Comment:
i would encourage you to use `classList.add()` only in case if:
- you reset element's class 
- reassign it to a list of classes while walking along some logic in the code.
example: 
```js
const div = document.createElement("div");

if (....) {
   div.classList.add("...");
}

if (....) {
   div.classList.add("...");
}

if (....) {
   div.classList.add("...");
}
```

in our case, coz as per our design there is only one class required for this element, i'd suggest you to do it this way:
```js
div.className = "episode";
```